be1-go/Makefile

```diff
@@ -9,7 +9,7 @@ lint:
 	staticcheck ./...
 
 check-cov: protocol lint vet
-	go test -v ./...
+	go test -v -race ./...
```

Comment:
I don't understand this makefile. There should be a 

```
check: test test-cov lint vet

test: protocol
	go test -v -race ./...

test-cov: protocol
	go test -v -coverpkg=./... -coverprofile=coverage.out ./... -json > report.json
```

And the CI would only do `make check`. Then we can remove the actual `check` and `check-cov`.